src/Parsers/Access/ParserCreateUserQuery.cpp

```diff
@@ -415,7 +420,8 @@ bool ParserCreateUserQuery::parseImpl(Pos & pos, ASTPtr & node, Expected & expec
 
         if (alter)
         {
-            if (new_name.empty() && (names->size() == 1) && parseRenameTo(pos, expected, new_name))
+            String maybe_new_name;
```

Comment:
Unused ?

Reply:
Yup, it's from the original PR: https://github.com/ClickHouse/ClickHouse/pull/37303/files#diff-ac49874c7389c62e3a512bdfb72dfd564c1f881ec2f9ec2739b3614e38f73c14R495, I can delete it here.